.github/workflows/zig.yml

```diff
@@ -0,0 +1,25 @@
+name: CI - Zig
+
+on:
+  create:
+  push:
+    branches: master
+  pull_request:
+  schedule:
+    - cron: "0 18 * * *"
```

Comment:
Why does this have a cron job? The zlib version will never change unless if the build.zig.zon is manually updated.